js/audio/player/AudioPlayer.svelte

```diff
@@ -44,6 +47,8 @@
 	let audio_player: HTMLAudioElement;
 
 	let stream_active = false;
+	let subtitles_toggle = false;
```

Comment:
Thoughts on displaying the subtitles by default if they are provided by the developer?

Reply:
Makes sense to me